src/coreclr/jit/importercalls.cpp

```diff
@@ -4415,28 +4371,26 @@ GenTree* Compiler::impSRCSUnsafeIntrinsic(NamedIntrinsic        intrinsic,
         }
 
         case NI_SRCS_UNSAFE_Write:
+        case NI_SRCS_UNSAFE_WriteUnaligned:
         {
             assert(sig->sigInst.methInstCount == 1);
 
             // ldarg.0
             // ldarg.1
+            // if NI_SRCS_UNSAFE_WriteUnaligned: unaligned. 0x01
             // stobj !!T
             // ret
 
-            return nullptr;
-        }
-
-        case NI_SRCS_UNSAFE_WriteUnaligned:
```

Comment:
Can we now remove these from VM?

Reply:
They're still used for indirect calls for now, the rest intrinsics are still there as well. I'd just remove them all at once when all are made JIT intrinsics.